docs/api/rest/health.md

```diff
@@ -0,0 +1,47 @@
+# Health APIs
+
+The Clara DICOM Adapter supports the following RESTful APIs on (default) port 5000.
+
+## GET /health/status
+
+DICOM Adapter service status:
+
+- Active DICOM DIMSE associations
+- Internal service status
+
+### Parameters
+
+N/A
+
+### Responses
+
+Response Content Type: JSON - [HealthStatusResponse](xref:Nvidia.Clara.DicomAdapter.API.Rest.HealthStatusResponse).
+
+| Code | Description          |
+| ---- | -------------------- |
+| 200  | Status is available. |
+| 500  | Server error.        |
```

Comment:
Are you following the mistake in the ACR health API? Does this mean RESTful Web server internal error, or the actual error of the DA?
Per http status use guidance, application service error should be returned in the message body with defined error code and message, instead of using the 500.
ACR API must also change too.

Reply:
Any error that occurred as the result of the action of the API call is included in the body along with the error code.